system/doc/efficiency_guide/profiling.xml

```diff
@@ -135,7 +135,7 @@
 
     <p>For a large system, you do not want to run the profiling
       tools on the whole system. Instead you want to concentrate on
-      central processes and modules, which contribute for a big part
+      central processes and modules, which contribute to a big part
       of the execution.</p>
```

Comment:
I don't think "contribute" is the right word here. Suggestion: "account for".

Reply:
I agree, will change now.